### ISSUE TITLE
"raise_if_conflicts" Error while using CocoaPods.

### ISSUE THREAD

**Issue by Vishaliphone** [CocoaPods/CocoaPods#4095]
Hi, My Mac OS X version is 10.10.3.
I had successfully installed cocoa pods on my mac.
While installing i got below error : 
 ERROR:  While executing gem ... (Gem::DependencyError)
    Unable to resolve dependencies: cocoapods requires nap (~> 0.8); cocoapods-trunk requires nap (~> 1.0); cocoapods-stats requires nap (~> 0.8); cocoapods-plugins requires nap (>= 0); cocoapods-core requires nap (~> 0.8.0)
So  did insalled nap (~> 0.8) &  nap (~> 1.0) by below command :

sudo gem install nap-v 0.8.0
sudo gem install nap -v 1.0

Then i install cocoa pods  & it's successfully installed in my mac.
Then i try pod help Command  to check is that installed properly. But i get below Error :-1: 

```
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/dependency.rb:296:in `to_specs': Could not find 'nap' (~> 0.8.0) - did find: [nap-1.0.0] (Gem::LoadError)
	from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/specification.rb:1209:in `block in activate_dependencies'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/specification.rb:1198:in `each'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/specification.rb:1198:in `activate_dependencies'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/specification.rb:1180:in `activate'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/specification.rb:1212:in `block in activate_dependencies'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/specification.rb:1198:in `each'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/specification.rb:1198:in `activate_dependencies'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/specification.rb:1180:in `activate'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_gem.rb:48:in `gem'
	from /usr/bin/pod:22:in `<main>'
```

Any help??

**Comment by segiddins** [comment]
This is not a complete bug report -- can you please follow the steps listed in https://github.com/CocoaPods/CocoaPods/blob/master/CONTRIBUTING.md ?

**Comment by segiddins** [comment]
Ah damn... You can uninstall the latest version of cocoapods-trunk and install the previous one, or you can use 0.39.0.beta.3

**Comment by Vishaliphone** [comment]
Thanks , Done.
I had Uninstalled "cocoapods-trunk" latest Version & Installed old one.
That salved my issue.

**Comment by segiddins** [comment]
This should now be fixed with the release of `cocoapods-trunk (0.6.4)`.
